.eslintrc.js

```diff
@@ -102,5 +102,88 @@ module.exports = {
         'valid-typeof': 'off',
       },
     },
-  ],
-}
+    {
+      files: [
+        "lib/index-bundle.ts",
```

Comment:
could we not use a glob like `lib/**/*.ts`?

Reply:
If we reformat the old files, then yes.

Reply:
Ok, this should be cleaned up later, than.